Sources/TuistCache/ContentHashing/GraphContentHasher.swift

```diff
@@ -94,6 +94,13 @@ public final class GraphContentHasher: GraphContentHashing {
             visited[target] = false
             return false
         }
+
+        // Ignore bundles becase they can not be cached
+        if target.target.product == .bundle {
```

Comment:
Should I pass this as a `IgnoreFilter` ? Then the `CacheGraphContentHasher` could pass it, so we keep the GraphContentHasher agnostic from cache logic.

Reply:
Yes, I think that's the proper way to do this